src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
@guisouza Yeah, `getContext` returns aways a new instance, so we have all methods as any instance. But we can discuss if when we call `getContext`, we return this new instance without `CONTEXTS` attribute ex.

    .getContext(context)//it works
    .getContext(context).getContext(context')//context does not exists